src/test/java/com/github/underscore/lodash/LodashTest.java

```diff
@@ -600,6 +600,16 @@ public void jsonToXml() {
             U.chain("{\n  \"a\": {\n  }\n}").jsonToXml().item());
         assertEquals("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<root empty-array=\"true\"></root>",
             U.jsonToXml("[]"));
+        assertEquals("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "<root>\n" +
+                "  <a>\n" +
+                "    <b>v1</b>\n" +
+                "  </a>\n" +
+                "  <c>v1</c>\n" +
+                "  <c>v2</c>\n" +
+                "  <c>v3</c>\n" +
```

Comment:
`+ ""` should be at the begin of string.

Reply:
Done.